app/scripts/views/base.js

```diff
@@ -103,7 +103,11 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
       this.fxaClient = options.fxaClient;
       this._canGoBack = options.canGoBack;
 
-      Backbone.View.call(this, options);
+      try {
+        Backbone.View.call(this, options);
+      } catch (e) {
+        console.error(e);
```

Comment:
Great start and thanks for looking into fixing this. we might need to report the exception to Sentry with `Raven.reportException`. Also, should this redirect to "unexpected error" as well?